rc/core/python.kak

```diff
@@ -47,19 +47,85 @@ add-highlighter -group /python/comment       fill comment
         add-highlighter -group /python/code regex '\b(${functions})\b\(' 1:builtin
     "
 
     # Highlight types and attributes
     printf %s "
-        add-highlighter -group /python/code regex '\b(${types})\b' 0:type
+        add-highlighter -group /python/code regex '\b(${types})\b' 1:type
         add-highlighter -group /python/code regex '@[\w_]+\b' 0:attribute
     "
 }
 
+# extended syntax
+
+%sh{
+    values="self"
+    # curl -s "https://docs.python.org/3/reference/datamodel.html#special-method-names" | ag -o '__[a-z_]+__[^(]' | sed s'/.$//' | sort | uniq
+    attributes="__abs__|__add__|__aenter__|__aexit__|__aiter__|__and__|__anext__"
+    attributes="${attributes}|__annotations__|__await__|__bases__|__bool__|__bytes__"
+    attributes="${attributes}|__call__|__class__|__classcell__|__closure__|__code__"
+    attributes="${attributes}|__complex__|__contains__|__defaults__|__del__"
+    attributes="${attributes}|__delattr__|__delete__|__delitem__|__dict__|__dir__"
+    attributes="${attributes}|__divmod__|__doc__|__enter__|__eq__|__exit__|__file__"
+    attributes="${attributes}|__float__|__floordiv__|__format__|__func__|__future__"
+    attributes="${attributes}|__ge__|__get__|__getattr__|__getattribute__"
+    attributes="${attributes}|__getitem__|__globals__|__gt__|__hash__|__iadd__"
+    attributes="${attributes}|__iand__|__ifloordiv__|__ilshift__|__imatmul__"
+    attributes="${attributes}|__imod__|__import__|__imul__|__index__|__init__"
+    attributes="${attributes}|__init_subclass__|__instancecheck__|__int__|__invert__"
+    attributes="${attributes}|__ior__|__ipow__|__irshift__|__isub__|__iter__"
+    attributes="${attributes}|__itruediv__|__ixor__|__kwdefaults__|__le__|__len__"
+    attributes="${attributes}|__length_hint__|__lshift__|__lt__|__matmul__"
+    attributes="${attributes}|__missing__|__mod__|__module__|__mro__|__mul__"
+    attributes="${attributes}|__name__|__ne__|__neg__|__new__|__next__|__objclass__"
+    attributes="${attributes}|__or__|__pos__|__pow__|__prepare__|__qualname__"
+    attributes="${attributes}|__radd__|__rand__|__rdivmod__|__repr__|__reversed__"
+    attributes="${attributes}|__rfloordiv__|__rlshift__|__rmatmul__|__rmod__"
+    attributes="${attributes}|__rmul__|__ror__|__round__|__rpow__|__rrshift__"
+    attributes="${attributes}|__rshift__|__rsub__|__rtruediv__|__rxor__|__self__"
+    attributes="${attributes}|__set__|__setattr__|__setitem__|__set_name__"
+    attributes="${attributes}|__slots__|__str__|__sub__|__subclasscheck__"
+    attributes="${attributes}|__truediv__|__weakref__|__xor__"
+    # curl -s "https://docs.python.org/3/reference/datamodel.html#special-method-names" | ag -o '__[a-z_]+__\| | sort | uniq
+    methods="__add__|__anext__|__await__|__bool__|__call__|__contains__|__del__"
+    methods="${methods}|__delattr__|__delete__|__delitem__|__divmod__|__enter__"
+    methods="${methods}|__eq__|__exit__|__floordiv__|__format__|__ge__|__get__"
```

Comment:
... and `__ge__` here.

The argument to `ag` seems incomplete, `'` is never matched (?)